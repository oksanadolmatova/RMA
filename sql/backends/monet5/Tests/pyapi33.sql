
START TRANSACTION;

CREATE FUNCTION pyapi33() RETURNS STRING LANGUAGE PYTHON { return 'hello'; };

SELECT pyapi33();

ROLLBACK;
