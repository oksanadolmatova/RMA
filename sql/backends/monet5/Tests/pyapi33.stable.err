stderr of test 'pyapi33` in directory 'sql/backends/monet5` itself:


# 14:44:15 >  
# 14:44:15 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=34589" "--set" "mapi_usock=/var/tmp/mtest-6105/.s.monetdb.34589" "--set" "monet_prompt=" "--forcemito" "--dbpath=/Users/myth/opt/var/MonetDB/mTests_sql_backends_monet5" "--set" "embedded_py=true"
# 14:44:15 >  

# builtin opt 	gdk_dbpath = /Users/myth/opt/var/monetdb5/dbfarm/demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 34589
# cmdline opt 	mapi_usock = /var/tmp/mtest-6105/.s.monetdb.34589
# cmdline opt 	monet_prompt = 
# cmdline opt 	gdk_dbpath = /Users/myth/opt/var/MonetDB/mTests_sql_backends_monet5
# cmdline opt 	embedded_py = true
# cmdline opt 	gdk_debug = 536870922

# 14:44:15 >  
# 14:44:15 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-6105" "--port=34589"
# 14:44:15 >  


# 14:44:16 >  
# 14:44:16 >  "Done."
# 14:44:16 >  

