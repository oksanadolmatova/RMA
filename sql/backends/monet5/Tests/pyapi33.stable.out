stdout of test 'pyapi33` in directory 'sql/backends/monet5` itself:


# 14:44:15 >  
# 14:44:15 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=34589" "--set" "mapi_usock=/var/tmp/mtest-6105/.s.monetdb.34589" "--set" "monet_prompt=" "--forcemito" "--dbpath=/Users/myth/opt/var/MonetDB/mTests_sql_backends_monet5" "--set" "embedded_py=true"
# 14:44:15 >  

# MonetDB 5 server v11.25.6
# This is an unreleased version
# Serving database 'mTests_sql_backends_monet5', using 4 threads
# Compiled for x86_64-apple-darwin15.6.0/64bit with 128bit integers
# Found 8.000 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2017 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://dhcp-46.eduroam.cwi.nl:34589/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-6105/.s.monetdb.34589
# MonetDB/SQL module loaded
# MonetDB/Python module loaded

Ready.
# SQL catalog created, loading sql scripts once
# loading sql script: 09_like.sql
# loading sql script: 10_math.sql
# loading sql script: 11_times.sql
# loading sql script: 12_url.sql
# loading sql script: 13_date.sql
# loading sql script: 14_inet.sql
# loading sql script: 15_querylog.sql
# loading sql script: 16_tracelog.sql
# loading sql script: 17_temporal.sql
# loading sql script: 18_index.sql
# loading sql script: 20_vacuum.sql
# loading sql script: 21_dependency_functions.sql
# loading sql script: 22_clients.sql
# loading sql script: 23_skyserver.sql
# loading sql script: 25_debug.sql
# loading sql script: 26_sysmon.sql
# loading sql script: 27_rejects.sql
# loading sql script: 39_analytics.sql
# loading sql script: 39_analytics_hge.sql
# loading sql script: 40_json.sql
# loading sql script: 40_json_hge.sql
# loading sql script: 41_md5sum.sql
# loading sql script: 45_uuid.sql
# loading sql script: 46_profiler.sql
# loading sql script: 51_sys_schema_extension.sql
# loading sql script: 72_fits.sql
# loading sql script: 74_netcdf.sql
# loading sql script: 75_storagemodel.sql
# loading sql script: 80_statistics.sql
# loading sql script: 80_udf.sql
# loading sql script: 80_udf_hge.sql
# loading sql script: 90_generator.sql
# loading sql script: 90_generator_hge.sql
# loading sql script: 99_system.sql

# 14:44:15 >  
# 14:44:15 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-6105" "--port=34589"
# 14:44:15 >  

#START TRANSACTION;
#CREATE FUNCTION pyapi33() RETURNS STRING LANGUAGE PYTHON { return 'hello'; };
#SELECT pyapi33();
% .L2 # table_name
% L2 # name
% clob # type
% 5 # length
[ "hello"	]
#ROLLBACK;

# 14:44:16 >  
# 14:44:16 >  "Done."
# 14:44:16 >  

